bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/PersistenceItem.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence;
+
+public interface PersistenceItem {
+    String getName();
+
+    Integer getRows();
```

Comment:
You are right, that rows is used by SQL databases. IIRC it differs using NOSQL databases.
The interface is about persistence, not about SQL databases.
If rows or files or map entries are used is implementation dependent.